operators/src/engine/result_descriptor.rs

```diff
@@ -47,6 +49,8 @@ pub struct RasterResultDescriptor {
     pub spatial_reference: SpatialReferenceOption,
     pub measurement: Measurement,
     pub no_data_value: Option<f64>,
+    pub time: Option<TimeInterval>,
+    pub bbox: Option<SpatialPartition2D>,
```

Comment:
yes, because we are planning to change the partitions to be expressed in pixels and I wanted to output data bounds in real world coordinates. But it's a fair point. I'm not sure if it's the right choice.